test/test_linalg.py

```diff
@@ -674,6 +675,122 @@ def test_norm_fastpaths(self, device):
         expected = torch.pow(x.pow(3).abs().sum(1), 1.0 / 3.0)
         self.assertEqual(result, expected)
 
+    @skipCPUIfNoLapack
+    @skipCUDAIfNoMagma
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @dtypes(torch.double, torch.float)
+    def test_eig_basic(self, device, dtype):
+        a = torch.Tensor(((1.96, 0.00, 0.00, 0.00, 0.00),
```

Comment:
`torch.tensor` instead of `Tensor`, instead of using `to()` you can pass the `dtype` and `device` kwargs to the `tensor()` constructor directly

Reply:
uh, you are correct. This is a test which was already there, I don't know why they wrote it like this in the first place.
Fixed by 63ca2b160d